package.json

```diff
@@ -44,6 +55,10 @@
     "immutable": "^3.7.5",
     "istanbul": "^0.4.3",
     "jscs": "^2.5.0",
-    "mocha": "^2.4.5"
+    "mocha": "^2.4.5",
+    "typescript": "^1.8.10",
+    "typings": "^1.0.4",
+    "tslint": "*",
+    "tslint-config-typings": "^0.2.0"
```

Comment:
That's quite a lot of dependencies added here. Could you add a quick comment about why you use them all for?

Is `typescript` actually necessary in the devDependencies? I'd be annoyed if I had to install an extra 4 dependencies (~40M) to use this small plugin on a JS package.

While we're here, can you bump versions of these dependencies and make sure it still works?